src/opamPrinter.mli

```diff
@@ -84,8 +101,10 @@ module Preserved : sig
       elements. The function assumes that [str] parses to the items
       [orig_its]. *)
   val items: string -> opamfile_item list -> opamfile_item list -> string
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Preserved.items instead."]
 
   val opamfile: ?format_from:file_name -> opamfile -> string
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Preserved.opamfile instead."]
   (** [opamfile f] converts [f] to string, respecting the layout and comments in
       the corresponding on-disk file for unmodified items. [format_from] can be
       specified instead of using the filename specified in [f]. *)
```

Comment:
GitHub won't let me make the full suggestion here - but the same as with `Normalise`, just deprecate the entire module.